### PR TITLE
To show recently completed picklist in store (#1x699zj)

### PR DESCRIPTION
**Pull request by bashu22tiwari**

### CODE REVIEW

File: src/views/Picklists.vue [hunk 1]
```diff
@@ -12,6 +12,16 @@
     </ion-header>
     
     <ion-content :fullscreen="true">
+      <ion-list v-if="completedPicklist.length">
+        <ion-list-header lines="none">
+          <ion-label>{{ $t("Completed") }}</ion-label>
```
Comment (ymaheshwari1) [inline src/views/Picklists.vue:17]: Internationalize the text.

**Review by ymaheshwari1 — CHANGES_REQUESTED** [review]

File: src/views/Picklists.vue [hunk 2]
```diff
@@ -12,6 +12,16 @@
...
+        </ion-list-header>
+        <!-- TODO : We are only storing the id of completed picklist, so not showing any other information right now -->
+        <ion-item v-for="picklist in completedPicklist" :key="picklist.picklistId">
```
Comment (ymaheshwari1) [inline src/views/Picklists.vue:20]: I think in completedPicklist, we are directly storing the ids, so the `:key` becomes invalid here, WDYT?

**Review by ymaheshwari1 — CHANGES_REQUESTED** [review]

File: src/views/Picklists.vue (hunk 1 above)
Comment (bashu22tiwari) [reply src/views/Picklists.vue:17]: Sir, Completed was already there in the en.json file.
Thank you Sir for reviewing my code.

File: src/views/Picklists.vue (hunk 2 above)
Comment (bashu22tiwari) [reply src/views/Picklists.vue:20]: Yes Sir, We can also change it with `:key="picklist"` .

File: src/views/Picklists.vue (hunk 1 above)
Comment (ymaheshwari1) [reply src/views/Picklists.vue:17]: Okay

File: src/views/Picklists.vue [hunk 3]
```diff
@@ -43,11 +53,14 @@ export default defineComponent({
     IonPage,
     IonTitle,
     IonToolbar,
-    PicklistItem
+    PicklistItem,
+    IonItem
```
Comment (ymaheshwari1) [inline src/views/Picklists.vue:57]: Arrange components alphabetically.

**Review by ymaheshwari1 — CHANGES_REQUESTED** [review]

File: src/views/Picklists.vue (hunk 3 above)
Comment (bashu22tiwari) [reply src/views/Picklists.vue]: Arranged components alphabetically.
Thank you sir for reviewing my code.

**Review by ymaheshwari1 — APPROVED** [review]

File: src/store/modules/picklist/mutation-types.ts [hunk 4]
```diff
@@ -1,3 +1,5 @@
 export const SN_PICKLIST = 'picklist'
 export const PICKLISTS_UPDATED = SN_PICKLIST + '/UPDATED'
 export const PICKLIST_CURRENT_UPDATED = SN_PICKLIST + '/CURRENT_UPDATED'
+export const PICKLIST_COMPLETED = SN_PICKLIST + '/COMPLETED'
```
Comment (adityasharma7) [inline src/store/modules/picklist/mutation-types.ts:4]: I think we could create a single mutation and add completed picklist in action, similarly we could reset items in actions and use same mutation 
```suggestion
export const PICKLIST_CMPLTD_UPDATED = SN_PICKLIST + '/CMPLTD_UPDATED'
```

Comment (adityasharma7) [reply src/store/modules/picklist/mutation-types.ts:4]: Also, we should store it with respect to logged in user. As there may be chances the logged in user is different. WDYT?

**Review by adityasharma7 — CHANGES_REQUESTED** [review]

File: src/store/modules/picklist/mutation-types.ts (hunk 4 above)
Comment (bashu22tiwari) [reply src/store/modules/picklist/mutation-types.ts]: Sir, Created a single mutation for adding the newly completed picklist and clearing the completed picklist.
Also, Sir, as if now we are having a single completed picklist for all the facilities of a single user, and clearing it once the user is logged out.
Thank you sir for reviewing my code.

**Pull request closed without merging**